### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by geoffdavis92** [dlindahl/gatsby-plugin-react-a11y#2]
I am interested in baking in accessibility into my Gatsby site, but it appears this plugin does not have any documentation.

Perhaps it would be useful to provide some so potential users could see if it is right for them and how to use it.

Thanks!
